app/src/main/java/org/coepi/android/ui/onboarding/OnboardingFragment.kt

```diff
@@ -17,4 +19,12 @@ class OnboardingFragment : Fragment() {
         lifecycleOwner = viewLifecycleOwner
         vm = viewModel
     }.root
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        if( Build.VERSION.SDK_INT >= 24 ) {
```

Comment:
This just leaves a white background when you're on a device less than 24. Can we put a different color in or something?

Reply:
Moved the background_gradient.xml to the main drawable folder instead of the api level specific one.